text/src/main/scala/org/apache/pekko/stream/connectors/text/javadsl/TextFlow.scala

```diff
@@ -45,9 +42,6 @@ object TextFlow {
    * Translates a stream of bytes from one character encoding into another.
    */
   def transcoding(incoming: Charset, outgoing: Charset): Flow[ByteString, ByteString, NotUsed] =
-    pekko.stream.scaladsl
-      .Flow[ByteString]
-      .via(new CharsetTranscodingFlow(incoming, outgoing))
-      .asJava
```

Comment:
I have to say, they are same code after all

Reply:
thanks, I update it